src/WebJobs.Extensions.Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -339,6 +348,14 @@ internal static void ApplyBindingData(object target, IDictionary<string, object>
                     bindingData[HttpHeadersKey] = request.Headers.ToDictionary(kvp => kvp.Key, kvp => kvp.Value.ToString(), StringComparer.OrdinalIgnoreCase);
                 }
 
+                if (!bindingData.ContainsKey(IdentitiesKey))
```

Comment:
Did you intend to check `isEasyAuthEnabled` here? I don't see that being used anywhere.

Reply:
This is now checked in the functions host.